lib/test_prof/recipes/rspec/let_it_be.rb

```diff
@@ -38,14 +38,15 @@ def modifiers
       end
 
       def wrap_with_modifiers(mods, &block)
-        return block if mods.empty?
+        # return block if mods.empty?
+        mods = {freeze: true}.merge(mods)
 
         validate_modifiers! mods
 
         -> {
           record = instance_eval(&block)
           mods.inject(record) do |rec, (k, v)|
-            LetItBe.modifiers.fetch(k).call(rec, v)
+            LetItBe.modifiers.fetch(k).call(rec, v, mods)
```

Comment:
Let's not allow modifiers to know about each other.

We can add `frozen?` checks to `reload` and `refind` modifiers instead. In case both `reload: true` and `freeze: true` are present in this order, it won't hurt (and there is a theoretical situations when it could be useful: for example, if you change the record state from the _outside_, smth like `context; let_it_be {}; context; before_all { change_underlying_db_row }; end; end;`).

Reply:
> We can add frozen? checks to reload and refind modifiers instead.

The problem is not that they will attempt reloading a frozen model, but that the model that will be reloaded were made frozen. So the stoplist is to keep track of those with `reload: true`/`refind: true`.

> theoretical situations when it could be useful

But how it will be reloaded if it's frozen, this will cause an `FrozenError`, and there's no way to unfreeze it.